### DISCUSSION TITLE
Does it support MacOS?

### DISCUSSION THREAD

**Discussion by HongkyVietnamese** [gri573/rethinking-voxels#320] [Q&A] [answered]
I'm playing minecraft on my M2 macbook. Can I use Rethinking Voxels?

**Comment by Aces-and-Jacks** [comment, marked as answer]
With any latest release it'll never be compatible. This is because `r0.1-alpha1a` does 

> bump openGL version from 330 to 430 for compute shader support (this breaks MacOS compatibility)


The last compatible version is `[Last OptiFine-Compatible Version] Beta 18d` which was before this update. If you really want to you'd have to lower your Iris version
